pyroomacoustics/tests/test_dirpat_class.py

```diff
@@ -0,0 +1,114 @@
+import numpy as np
```

Comment:
The test is not testing anything. It should have at least one method inside that is prefixed with `test_`. Such methods are run automatically. In the method we check that values returned by the new functions/code match some expected values. The check is done with an assert that should fail if an unexpected value is returned.